app/components/section-menu/component.js

```diff
@@ -48,10 +48,11 @@ export default Ember.Component.extend({
                 return {
                     section: section,
                     label: section["section-header"],
-                    id: "section-" + index
-                }
+                    id: "section-" + index,
+                    nav: section["settings.show-on-nav"]
+                };
             })
-            .filter((menuItem, index) => index > this.get("index"));
+            .filter((menuItem) => menuItem.section['settings']['show-in-nav']);
```

Comment:
maybe "anchored" ?

Reply:
non-technical users might not understand what that means

Reply:
this also is a whitelist, what would the benefits/negatives be of using a blacklist? First thing that comes to ming is a non-tech user puts this component in, and they see nothing because everything is blacklisted by default

Reply:
super good point